modules/sr/robot/motor.py

```diff
@@ -76,15 +74,12 @@ def power(self, value: int) -> None:
 
         self.sr_motor.set_speed(translate(value, self.sr_motor))
 
-    ''''@property
-    def use_brake(self) -> bool:
-        "Whether to use the brake when at 0 speed"
-        return self._use_brake
+    @property
+    def use_brake(self):
```

Comment:
Not sure if your branch contains the latest from master -- we require type annotations everywhere, so we'll need one here. I think it should just be `bool`, but do check the docs.
(The setter below may also need one too).